build/pkgs/gfortran/distros/macports.txt

```diff
@@ -1 +1,2 @@
-gcc10 +gfortran
+gcc10
++gfortran
```

Comment:
What's that `+` there?

Reply:
I don't remember, but I'm pretty sure it's not a typo. We can revisit this if we ever get #35667 unstuck